program/src/state.rs

```diff
@@ -2383,3 +2416,23 @@ impl AdvancedOrders {
         Ok(state)
     }
 }
+
+#[derive(Copy, Clone, Pod, Loadable)]
+pub(crate) struct TokenAccount(pub(crate) Account);
+
+impl TokenAccount {
```

Comment:
maybe add a `pub fn account()/data() -> &Account`, to avoid having to use `.0` externally?